tests/test_phase8_a1_domain_primitives_entitlement.py

```
@@ -0,0 +1,729 @@
+            # Create PERK event
+            ent_id = "ent-perk-active"
+            _add_entitlement_event_for_test(
+                event_id=str(uuid.uuid4()),
+                entitlement_id=ent_id,
+                target_seat_id=seat_id,
+                class_id=class_id,
+                entitlement_type="HALL_PASS",
+                event_type="GRANTED",
+                acquisition_type="PERK",
+                product_id=101,
+                timestamp=datetime.utcnow(),
```

Ruff (0.16.0)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)